schema.sql

```diff
@@ -10,3 +10,26 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals ADD COLUMN species varchar(40);
+
+
+CREATE TABLE owners (
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	full_name VARCHAR (50) NOT NULL,
+	age INT NOT NULL
+);
+
+CREATE TABLE species(
+		id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+		name VARCHAR(50) NOT NULL
+);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD species_id INT REFERENCES species(id);
+
+ALTER TABLE animals ADD CONSTRAINT species_id FOREIGN KEY (species_id) REFERENCES species(id);
+
+ALTER TABLE animals ADD owner_id INT REFERENCES owners(id);
```

Comment:
- [ ] Here as well I think you don't need to make alter here you just want to add foreignkey on species and oweners so make sure if you need this and if you don't kindly remove it 🚀💯